actors/miner/src/monies.rs

```diff
@@ -267,10 +271,40 @@ pub fn initial_pledge_for_power(
     let lock_target_denom = LOCK_TARGET_FACTOR_DENOM;
     let pledge_share_num = qa_power;
     let network_qa_power = network_qa_power_estimate.estimate();
-    let pledge_share_denom = cmp::max(cmp::max(&network_qa_power, baseline_power), qa_power);
+
+    // Once FIP-0081 has fully activated, additional pledge will be 70% baseline
+    // pledge + 30% simple pledge.
+    const FIP_0081_ACTIVATION_PERMILLE: i64 = 300;
+    // Gamma is the share of pledge coming from the baseline formulation, with 1-gamma coming from simple pledge.
+    // Gamma ranges from 1.0 before the ramp to 0.7 at the end.
+    // gamma = 1000 - 300 * (epochs_since_ramp_start / ramp_duration_epochs).max(0).min(1)
+    let skew = if epochs_since_ramp_start < 0 {
+        // No skew before ramp start
+        0
+    } else if ramp_duration_epochs == 0 || epochs_since_ramp_start >= ramp_duration_epochs as i64 {
+        // 100% skew after ramp end
+        FIP_0081_ACTIVATION_PERMILLE as u64
+    } else {
+        ((epochs_since_ramp_start * FIP_0081_ACTIVATION_PERMILLE) / ramp_duration_epochs as i64)
+            as u64
+    };
+    let gamma = FIXED_POINT_FACTOR - skew;
+
     let additional_ip_num = lock_target_num * pledge_share_num;
-    let additional_ip_denom = pledge_share_denom * lock_target_denom;
-    let additional_ip = additional_ip_num.div_floor(&additional_ip_denom);
+
+    let pledge_share_denom_baseline =
+        cmp::max(cmp::max(&network_qa_power, baseline_power), qa_power);
+    let pledge_share_denom_simple = cmp::max(&network_qa_power, qa_power);
+
+    let additional_ip_denom_baseline = pledge_share_denom_baseline * lock_target_denom;
+    let additional_ip_baseline = additional_ip_num.div_floor(&additional_ip_denom_baseline);
+    let additional_ip_denom_simple = pledge_share_denom_simple * lock_target_denom;
+    let additional_ip_simple = additional_ip_num.div_floor(&additional_ip_denom_simple);
+
+    // convex combination of simple and baseline pledge
+    let additional_ip = (additional_ip_baseline * gamma
+        + additional_ip_simple * (FIXED_POINT_FACTOR - gamma))
+        / FIXED_POINT_FACTOR;
```

Comment:
Do you know what the difference is between `/` and `div_floor` ?  I can look into it but if you have a good reason for using `/` here it would help me to know your thinking.

Reply:
They should be the same since all the numbers are positive. The division has been removed, though.